libs/designsystem/src/lib/components/page/page.component.scss

```diff
@@ -65,10 +66,27 @@ ion-title {
 }
 
 ion-back-button {
+  @include ionic.apply-hover('s');
+
   --color: #{utils.get-color('black')};
   --icon-font-size: 24px;
+
   height: utils.$fat-finger-size;
   width: utils.$fat-finger-size;
+
+  // NOTE: Also see ion-segment-button in segmented control component
```

Comment:
Same point as above comment regarding references to page file and this one.

Reply:
Same answer as above 🙂